packages/plugins/ToggleSideBars/src/index.ts

```diff
@@ -2,6 +2,7 @@ import joplin from 'api';
 
 joplin.plugins.register({
 	onStart: async function() {
-		console.info('Test plugin started!');
+		await joplin.views.toolbarButtons.create('tsb', 'toggleSideBar', 'noteToolbar');
+		await joplin.views.toolbarButtons.create('tnl', 'toggleNoteList', 'noteToolbar');
```

Comment:
As the ID are globally unique, it should be something longer, such as "toggleSideBarButton" and "toggleNoteListButton".

Reply:
Sure, makes sense. Let me change that quickly.